pom.xml

```diff
@@ -635,6 +640,19 @@ under the License.
     </dependencies>
     <!--bootstrap-start-comment-->
   </dependencyManagement>
+
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <id>sonatype-snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
```

Comment:
I am pretty sure this causes the IT failures, because some ITs seem to rely on dynamic versions and check on the wrong repo now (which delivers an outdated result), however I see no artifact being resolved from `sonatype-snapshots` in https://github.com/apache/maven/actions/runs/9322027451/job/25662372946?pr=1546.

Reply:
Seems the IT failure was fixed yesterday in https://github.com/apache/maven-integration-testing/commit/4d1c9987e04b2581027f8c1c6645212c7e1f24b8.